src/CollarProviderNFT.sol

```diff
@@ -42,6 +42,7 @@ contract CollarProviderNFT is ICollarProviderNFT, BaseNFT {
     using SafeERC20 for IERC20;
 
     uint internal constant BIPS_BASE = 10_000;
+    uint internal constant YEAR = 365 days;
```

Comment:
I think what the auditor meant was to use the reserved word `years` not a variable , if im nopt mistaken